projects/Deagle.md

```diff
@@ -0,0 +1,65 @@
+## Project Name 
+Deagle
+
+## Category 
+Application
+
+## Project Description
+Deagle is an NFT options protocol. Using deagle, a user can speculate, hedge and earn income on his/her NFT. Instead of trading NFTs, trade the option to buy or sell an NFT at a strike price before a maturity date. NFT options data is stored in IPFS.
+There will be two types of user, 
+Option Seller : He is the person who has the nft so he can either mint a put option to hedge his/her nft(we call it NFT protection) or mint a call option to speculate and earn income on his/her NFT.
+Option buyer: He is the person who is going to buy an option from any secondary marketplace and can settle it on our application if his option expires in the money. 
+
+## Use of IPFS, Filecoin and Libp2p
+NFT options metadata is stored in IPFS. This metadata includes option strike price, option expiry timestamp and long short pair address (unique for a set of strike price, expiry date and underlying nft, required for option settlement). 
+
+## Project Status
+Beta
+
+## Previews
+<img width="943" alt="Screenshot 2021-08-16 at 2 11 57 AM" src="https://user-images.githubusercontent.com/24940106/130564008-fbedf722-e388-4625-a31b-14a9a247d30c.png">
+<img width="726" alt="Screenshot 2021-08-16 at 1 40 04 AM" src="https://user-images.githubusercontent.com/24940106/130564097-492fb6c3-e4f3-4616-bd7c-6880188ed745.png">
+<img width="726" alt="Screenshot 2021-08-16 at 1 40 40 AM" src="https://user-images.githubusercontent.com/24940106/130564106-d942a22c-3227-4e00-a105-654a7da223f4.png">
+<img width="726" alt="Screenshot 2021-08-16 at 1 40 47 AM" src="https://user-images.githubusercontent.com/24940106/130564111-1f263ac8-90ea-42a5-8fdd-963aae13dfb4.png">
+<img width="726" alt="Screenshot 2021-08-16 at 1 41 12 AM" src="https://user-images.githubusercontent.com/24940106/130564119-4ffa34f4-4af2-460b-90c2-c193caebdb6b.png">
+
+
+
+
+## Target Audience
+Problems faced by NFT enthusiasts are:
+- Good NFTs are very expensive.
+- No income on idle NFTs.
+- No protection from the downside if the NFTs starts to lose its value.
+To take advantage of the high volatility in NFTs without needing lumpsum amount of money, Options would be a great fit. 
+
+
+## Rough estimated user base (if applicable)
+Application is just live on testnet. Planning to make it live soon as the demand for it seems to be rising.
+
+## Github repo
+https://github.com/i-m-aditya/Deagle
+
+## Website
+https://nft-options.netlify.app
+
+<!--If you're applying for a Next Step grant, add the URL to your hackathon submission here also-->
+
+## Docs
+<!--Including a link to your project docs!-->
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
+### Team Size
+Currently hacking solo, will be adding frontend engineers soon.
+
+### Team members  
```

Comment:
This field is blank, if you add yourself here, or remove the field, I think either would work :).